src/app/game.ts

```diff
@@ -112,6 +112,14 @@ export class Game {
     this.villageUnits.push(unit)
   }
 
+  possibleToAddUnit() {
+    return this.population > this.villageUnits.length && this.getGoldAmount() >= 2
```

Comment:
Such "magic" numbers cannot be used in code (2). When we change in the future the amount of gold needed to buy the unit then we have to change this number. It should be a constant set in the Warrior class and actually we have only warrior but we could have other units so this calculation won't be working

Reply:
pass unitCost as a param of this function and argument in here
![image](https://user-images.githubusercontent.com/38506213/106877531-259e4d80-66d9-11eb-8f6f-aaf55e595f3c.png)